### DISCUSSION TITLE
Inquiring the computational need of Medperf

### DISCUSSION THREAD

**Discussion by anushkai** [mlcommons/medperf#460] [Q&A] [answered]
I am planning to Experiment Medperf in AWS. But for that I need to estimate the resource consumption to reduce the costs. Can I get a rough estimation if someone is aware about the values?

**Comment by aristizabal95** [comment]
Hey @anushkai! MedPerf, by itself, doesn't have a certain criteria for resource consumption. Since MedPerf provides the means for others to create and manage benchmarking experiments, hardware requirements are defined by use case. A more valuable question would be, what are you planning to do with MedPerf at the moment? If you're looking into following the tutorials and running the basic examples, then you should be okay with pretty basic (most probably free tier) VMs. Other examples provided here, like the BraTS and Pancreas segmentation ones do require more computation, which if you're interested in I can help gather those requirements.

Let us know what you're looking for and we will be happy to help you!

**Comment by anushkai** [comment]
@aristizabal95 Thank you for the response. I am planning to do the experiments in phases. At beginning I will work with basic examples. After being familiarizedas the second phase I would move to complex examples such as Pancreas segmentation. So I think if I can have the resource requirements for such ones it would be more helpful.

**Reply by aristizabal95** [reply, marked as answer]
For the Pancreas Segmentation example we used a GCP VM with 2vCPU + 8GB memory (e2-standard-2), and a 256GB disk. I can't quite remember how long it took to process, but I'll ask for estimates from the model owner. I don't think it took more than a few hours per data split, so it shouldn't be too expensive to run.

Do let us know if you need help running those examples. I don't believe they are as well documented as everything else

**Reply by anushkai** [reply]
Thank you @aristizabal95 . I'll contact you if I had more problems.
